src/test/kotlin/com/exactpro/th2/common/schema/message/impl/rabbitmq/connection/TestConnectionManager.kt

```diff
@@ -824,29 +816,25 @@ class TestConnectionManager {
         )
 
     @Test
-    @Disabled
-    // TODO: this test is no more relevant
-    // TODO: we need to change test scenario or remove it
     fun `connection manager exclusive queue test`() {
-        RabbitMQContainer(RABBITMQ_IMAGE_NAME)
-            .use { rabbitMQContainer ->
-                rabbitMQContainer.start()
-                LOGGER.info { "Started with port ${rabbitMQContainer.amqpPort}" }
-
-                createConnectionManager(rabbitMQContainer).use { firstManager ->
-                    createConnectionManager(rabbitMQContainer).use { secondManager ->
-                        val queue = firstManager.queueDeclare()
-
-                        assertFailsWith<IOException>("Another connection can subscribe to the $queue queue") {
-                            secondManager.basicConsume(queue, { _, _, _ -> }, {})
-                        }
-
-                        extracted(firstManager, secondManager, queue, 3)
-                        extracted(firstManager, secondManager, queue, 6)
-                    }
+        RabbitMQContainer(RABBITMQ_IMAGE_NAME).use { rabbitMQContainer ->
+            rabbitMQContainer.start()
+            LOGGER.info { "Started with port ${rabbitMQContainer.amqpPort}" }
+
+            createConnectionManager(rabbitMQContainer).use { firstManager ->
+                createConnectionManager(rabbitMQContainer).use { secondManager ->
+                    val queue = firstManager.queueDeclare()
+                    val consumerThread = thread { secondManager.basicConsume(queue, { _, _, _ -> }, {}) }
+                    consumerThread.join(5_000)
+                    val isAlive = consumerThread.isAlive
+                    consumerThread.stop()
```

Comment:
The `stop` method is deprecated in Java. We probably should correct the approach for testing this...